public/GlueBackend/index.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use Pyz\Glue\GlueApplication\Bootstrap\GlueBackendApiBootstrap;
+use Spryker\Shared\Config\Application\Environment;
+use Spryker\Shared\ErrorHandler\ErrorHandlerEnvironment;
+
+define('APPLICATION', 'GLUE_BACKEND');
+defined('APPLICATION_ROOT_DIR') || define('APPLICATION_ROOT_DIR', dirname(__DIR__, 2));
+
+require_once APPLICATION_ROOT_DIR . '/vendor/autoload.php';
+
+Environment::initialize();
+
+$errorHandlerEnvironment = new ErrorHandlerEnvironment();
+$errorHandlerEnvironment->initialize();
+
+$bootstrap = new GlueBackendApiBootstrap();
+$bootstrap
+    ->boot()
```

Comment:
`->boot([BackendApiGlueApplicationBootstrapPlugin::class])` could be done here. This could held to get rid of `src/Pyz/Glue/GlueApplication/Bootstrap/GlueBackendApiBootstrap.php`.